contributing.md

```diff
@@ -47,6 +47,17 @@ and start hacking:
 $ git checkout -b my-feature-branch
 ```
 
+### PRE COMMIT
+
+We automatically run lint as our pre-commit command.
```

Comment:
we should run lint and test right?

Reply:
Talked offline. We will leave the test for now and let the team decide what to do